wesnoth-wof.po

```diff
@@ -2872,13 +2923,18 @@ msgid ""
 "clear.\n"
 "Those hatchlings will be slain if our intervention is not swift."
 msgstr ""
+"I když ten haldrik mluví mnoha neznámými slovy, jejich hrozivé vyznění je "
+"jasné.\n"
+"Pokud rychle nezasáhneme, budou tato vylíhlá mláďata zabita."
 
 #. [message]: speaker=Zedrix
 #: data/campaigns/Winds_of_Fate/scenarios/05_Threshold.cfg:728
 msgid ""
 "He returns! Noble Shek’kahan returns from ssspirit realm to sssave usssss! "
 "(<i>chirping sob</i>)"
 msgstr ""
+"Vrací sse! Vzneššený Ššekekahan se vrací zz duchovní říšše, aby násss "
+"zzachránil! (<i>Štěbetavě vzlyká.</i>)"
```

Comment:
Nestačilo by vysvětlení stejně prostě jako v originále (tedy ne úplnou větou s písmenem a tečkou)?
```suggestion
"zzachránil! (<i>štěbetavě vzlyká</i>)"
```

Reply:
Už jsem to tak udělal v některých jiných případech. Přijde mi, že se originál stejně postupně aktualizuje do toho, že to bude vyjádřeno jako úplná věta. Samozřejmě, můj odhad nemusí být správný, ale tuším, že to je dost trend, který jsme viděl i v jiných repositářích.
Navíc se mi to takhle víc líbí, neleží tam jen jakási useklá větev, ale „celá věta“.